app/controllers/users_controller.rb

```diff
@@ -1,5 +1,7 @@
 class UsersController < ApplicationController
   before_action :set_user, only: %i[ show liked feed followers following discover ]
+  before_action :authorize_user
```

Comment:
Ah, is this another way of doing the same thing I did in my controllers, but by defining a method to call instead?